tests/test_compile.py

```diff
@@ -28,7 +28,7 @@ def setUp(self):
         os.chdir(os.getcwd() + '/examples/kubernetes/')
 
     def test_compile(self):
-        sys.argv = ["kapitan", "compile", "-f"]
```

Comment:
I think we should have a test for the cache functionality as well.

Maybe 2 separate tests

- kapitan compile
- kapitan compile --cache

Reply:
Will write a test in another MR